### PR TITLE
Fixed broken link /martin-hinshelwood/

### PR DESCRIPTION
**Pull request by tiagov8**
> 1. What triggered this change? (PBI link, Email Subject, conversation + reason, etc)

CodeAuditor

> 2. What was changed?

Fixed link

> 3. Did you do pair or mob programming (list names)?

no

### CODE REVIEW

File: Martin-Hinshelwood/Martin-Hinshelwood.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ qualifications:  "BSc Computer Studies, Microsoft Certified Professional, Micros
...
 ---
 
-Martin is a Solution Architect at SSW, and also one of three [Microsoft Visual Studio ALM MVP](https://mvp.support.microsoft.com/profile/Martin.Hinshelwood) in the UK and has over 9 years experience in the software industry. He regularly writes on his [ALM blog](https://blog.hinshelwood.com/), has many articles published on [Code Project](https://www.codeproject.com/Members/Martin-Hinshelwood) and is a moderator on Microsoft forums. His expertise includes (but is not exclusive to) Team Foundation Server, Scrum, WPF, Silverlight, SharePoint, VB.NET, C#, WCF, ASP.NET. 
+Martin is a Solution Architect at SSW, and also one of three [Microsoft Visual Studio ALM MVP](https://mvp.support.microsoft.com/profile/Martin.Hinshelwood) in the UK and has over 9 years experience in the software industry. He regularly writes on his [ALM blog](https://blog.hinshelwood.com/), has many articles published on [Code Project](https://www.codeproject.com) and is a moderator on Microsoft forums. His expertise includes (but is not exclusive to) Team Foundation Server, Scrum, WPF, Silverlight, SharePoint, VB.NET, C#, WCF, ASP.NET.
```
Comment (jeoffreyfischer) [inline Martin-Hinshelwood/Martin-Hinshelwood.md:10]: https://mvp.support.microsoft.com/profile/Martin.Hinshelwood

This link is not working for me.. Can you double check?

Comment (jeoffreyfischer) [inline Martin-Hinshelwood/Martin-Hinshelwood.md:10]: https://blog.hinshelwood.com/

This link is broken

Comment (tiagov8) [reply Martin-Hinshelwood/Martin-Hinshelwood.md]: Confirmed, so I removed the link

PS. For this PR I just fixed the one found by CodeAuditor

Comment (tiagov8) [reply Martin-Hinshelwood/Martin-Hinshelwood.md]: Done - updated URL

**Comment by tiagov8** [comment]
Hi @jeoffreyfischer 

For this PR I just fixed the one link found by CodeAuditor. So thanks for testing the others ;)

I have now made further changes to this profile

**Review by jeoffreyfischer — APPROVED** [review]
LGTM

**Pull request merged**